src/ui/js/Game.js

```diff
@@ -2791,11 +2797,64 @@ Game.createGameMatDieDiv = function(die, player, dieStatus, isClickable) {
     dieNumberSpanOpts.html = '&nbsp;' + die.value + '&nbsp;';
   }
 
+  divOpts.style = 'background-image: ' + Game.backgroundImagePath(
+    die.sides,
+    dieStatus,
+    isClickable
+  );
+
   var dieDiv = $('<div>', divOpts);
   dieDiv.append($('<span>', dieNumberSpanOpts));
+
   return dieDiv;
 };
 
+/*
+ * Calculate the background image path
+ *
+ * @param  int     sides          Number of sides of the die
+ * @param  string  dieStatus      Status of the die ('active' or 'captured')
+ * @param  boolean isClickable    Is the die clickable?
+ * @return string
+ */
+Game.backgroundImagePath = function(sides, dieStatus, isClickable) {
+  var sidesRoundedUp;
+  var imageType;
+
+  if (sides <= 2) {
+    sidesRoundedUp = 2;
+  } else if (sides <= 4) {
+    sidesRoundedUp = 4;
+  } else if (sides <= 6) {
+    sidesRoundedUp = 6;
+  } else if (sides <= 8) {
+    sidesRoundedUp = 8;
+  } else if (sides <= 10) {
+    sidesRoundedUp = 10;
+  } else if (sides <= 12) {
+    sidesRoundedUp = 12;
+  } else if (sides <= 20) {
+    sidesRoundedUp = 20;
+  } else {
+    sidesRoundedUp = 30;
+  }
```

Comment:
Nit: this isn't so inelegant i'll reject it, but it is inelegant --- you could instead iterate over an array of rounded-up sizes and compare them to `sides`.  If you want to do that and only didn't because you didn't want to figure out the syntax, i can take a few minutes this evening and come up with it.